src/contracts/pods/EigenPodManager.sol

```diff
@@ -267,8 +259,42 @@ contract EigenPodManager is
         return (currentDepositShares < 0 ? 0 : uint256(currentDepositShares), shares);
     }
 
+    /// @dev Calculates the proportion a pod owner's restaked balance has decreased, and
+    /// reduces their beacon slashing factor accordingly.
+    /// Note: `balanceDecreasedWei` is assumed to be less than `prevRestakedBalanceWei`
+    function _reduceSlashingFactor(
+        address podOwner,
+        uint256 prevRestakedBalanceWei,
+        uint256 balanceDecreasedWei
+    ) internal returns (uint256, uint64, uint256) {
+        // Apply negative balance delta to calculate the proportion of the original
+        // balance that remains. Note that underflow here should be impossible given
+        // the invariants pods use to calculate these values.
+        uint256 newRestakedBalanceWei = prevRestakedBalanceWei - balanceDecreasedWei;
+        uint256 balanceRemainingWad = newRestakedBalanceWei.divWadRoundUp(prevRestakedBalanceWei);
```

Comment:
Nit: The naming here is throwing me off since I view this as a proportion, but balanceRemaining implies a delta. Is the thinking that Wad reflects this term as a proportion?

Reply:
> Is the thinking that Wad reflects this term as a proportion?

yes that was the idea. What about `proportionRemainingWad`?

Reply:
I like that one better :)

Reply:
[678abaf](https://github.com/Layr-Labs/eigenlayer-contracts/pull/892/commits/678abaf2c00c20254daf9b4515dca86cae2495d3)